Source Main 5.2/source/WSclient.cpp

```diff
@@ -6416,7 +6416,7 @@ void ReceiveLevelUp(const BYTE* ReceiveBuffer, int Size)
     }
 
     wchar_t szText[256] = { NULL, };
-    WORD iExp = CharacterAttribute->NextExperience - CharacterAttribute->Experience;
+    uint64_t iExp = CharacterAttribute->NextExperience - CharacterAttribute->Experience;
```

Comment:
Exception encountered when leveling up due to incorrect data type

Reply:
I think you can actually remove this 4 lines. There is no need to print out this exp gain, because the server sent a corresponding exp message before anyway.

Reply:
Agreed, I applied the suggestion and works fine.